<documentation title="Doc Comment Throws Tag">
    <standard>
    <![CDATA[
    If a function throws any exceptions, they should be documented in a @throws tag.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: @throws tag used.">
        <![CDATA[
/**
 * <em>@throws SyException all the time</em>
 * @return void
 */
function foo()
{
    throw new SyException('Danger!');
}
]]>
        </code>
        <code title="Invalid: No @throws tag used for throwing function.">
        <![CDATA[
/**
 * @return void
 */
function foo()
{
    throw new SyException('Danger!');
}
]]>
        </code>
    </code_comparison>
</documentation>
